conf/messages.en

```diff
@@ -920,7 +920,7 @@ estimatedIncome.checkYourAnswersLabel = Estimated income for the current 12 mont
 # What is the charity’s actual income so far?
 # ----------------------------------------------------------
 actualIncome.title = What is the charity’s actual income so far?
-actualIncome.heading = What is the charity’s actual income so far?
+actualIncome.heading = What is the charity's actual income in pounds so far?
```

Comment:
Maintain the correct apostrophe `’` and also adjust the title as the content should match.

You will see when you test that the current one used isn't displayed. Please provide screenshot for design sign-off too.

Reply:
Done